plugins/node/opentelemetry-instrumentation-knex/src/utils.ts

```diff
@@ -18,7 +18,7 @@ import { Exception } from '@opentelemetry/api';
 import {
   DB_SYSTEM_NAME_VALUE_POSTGRESQL,
   DB_SYSTEM_NAME_VALUE_SQLITE,
-} from './semconv';
+} from '@opentelemetry/semantic-conventions';
```

Comment:
To use `DB_SYSTEM_NAME_VALUE_SQLITE` will require using a local `src/semconv.ts` soon. See:
https://github.com/open-telemetry/opentelemetry-js/pull/5690#issue-3067435791

tl;dr: `DB_SYSTEM_NAME_VALUE_SQLITE` is actually *un*stable and should not have been included in the stable `@opentelemetry/semantic-conventions` entry point.

(Eventually semconv-related things will settle down and this won't be so hard/obtuse/subtle.)

Reply:
> To use `DB_SYSTEM_NAME_VALUE_SQLITE` will require using a local `src/semconv.ts` soon.

Perhaps easiest to wait for a v1.33.1 release of the semconv package, then create the src/semconv.ts again (perhaps using the https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/scripts/gen-semconv-ts.js tool). I'm happy to help with that.